docker/centos6-x64/Dockerfile

```diff
@@ -5,9 +5,36 @@ MAINTAINER Rapid7 Release Engineering <r7_re@rapid7.com>
 
 VOLUME /pkg
 
-RUN rpm -ivh http://download.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+# # Copy newer apt sources that are valid/use https
+COPY yum.repos.d /etc/yum.repos.d
+
+RUN rpm -ivh https://dl.fedoraproject.org/pub/archive/epel/6/x86_64/epel-release-6-8.noarch.rpm
 RUN yum upgrade -y && yum clean all
-RUN yum install -y centos-release-SCL && yum clean all
```

Comment:
I'm guessing this was needed for signing; Maybe I'll need to circle back to this